test/conftest.py

```diff
@@ -642,6 +643,26 @@ def indexing_document_classifier():
     )
 
 
+@pytest.fixture
+def transformers_query_classifier():
+    return TransformersQueryClassifier(
+        model_name_or_path="shahrukhx01/bert-mini-finetune-question-detection",
+        use_gpu=False,
+        task="text-classification",
+        labels=["LABEL_1", "LABEL_0"],
+    )
+
+
+@pytest.fixture
+def zero_shot_transformers_query_classifier():
+    return TransformersQueryClassifier(
+        model_name_or_path="typeform/distilbert-base-uncased-mnli",
+        use_gpu=False,
+        task="zero-shot-classification",
+        labels=["happy", "unhappy", "neutral"],
+    )
+
+
```

Comment:
Can you move these into the query classifier suite? 

----

_PS: I know, most other fixtures are here, but we're refactoring the tests and we will have to track down which tests uses which fixtures. Moving them already in the suites that uses them is a great help_ :slightly_smiling_face:

Reply:
Yes!